tools/secrets-tool/commandhelper.cpp

```diff
@@ -467,6 +468,54 @@ void CommandHelper::start(const QString &command, const QStringList &args)
         r->setKeyTemplate(keyTemplate);
         r->setKeyDerivationParameters(kdp);
 
+        m_cryptoRequest.reset(r);
+        m_cryptoRequest->setManager(&m_cryptoManager);
+        connect(m_cryptoRequest.data(), &Sailfish::Crypto::Request::statusChanged,
+                this, &CommandHelper::cryptoRequestStatusChanged);
+        m_cryptoRequest->startRequest();
+    } else if (command == QStringLiteral("--import-stored-key")) {
+        const QString importDataFile = args.value(4);
+        if (importDataFile.isEmpty()
+                || !QFile::exists(importDataFile)) {
+            qInfo() << "Invalid import data file specified!";
+            emitFinished(EXITCODE_FAILED);
+            return;
+        }
+
+        QFile importFile(importDataFile);
+        if (!importFile.open(QIODevice::ReadOnly)) {
+            qInfo() << "Unable to open import data file for reading!";
+            emitFinished(EXITCODE_FAILED);
+            return;
+        }
+
+        const QByteArray importData = importFile.readAll();
+        if (importData.isEmpty()) {
+            qInfo() << "Unable to read data from import file!";
```

Comment:
Re-word the error message. It is also possible that it could read data from the file, but the file was simply empty.

Reply:
Done